syntax = "proto3";

option java_package = "AnimFireChalBuilding";
option java_outer_classname = "AnimFireChalProto";

package AnimFireChalBuilding;

service AnimFireChalBuilding {
  rpc getBuildingInfo (Empty) returns (BuildingInfo) {}
  rpc getRewards (Empty) returns (Reward) {}
}

message BuildingInfo {
	repeated Building buildings = 1; 
}

message Building {
  int32 fieryness = 1;
  double temperature = 2;
  int32 building_id = 3;
}

message Empty {
}

message Reward {
	double reward = 1;
}